daemon/cmd/daemon.go

```diff
@@ -482,6 +482,7 @@ func NewDaemon(ctx context.Context, dp datapath.Datapath) (*Daemon, *endpointRes
 	// Trigger refresh and update custom resource in the apiserver with all restored endpoints.
 	// Trigger after nodeDiscovery.StartDiscovery to avoid custom resource update conflict.
 	if option.Config.IPAM == option.IPAMCRD || option.Config.IPAM == option.IPAMENI || option.Config.IPAM == option.IPAMAzure {
+		// Do we need to split here the IPAMENI since it only support IPv4
```

Comment:
While I am awaiting verification on whether ENI is only the mode that is supported with IPv4, I don't think we need any more checks here as this code will not be reached because we call `option.Config.Validate()` on line 258

Reply:
So far we only support IPv4 for ENI and Azure IPAMs, but it is only documented on ENI, so we don't need to split.